### PR TITLE
Update sdk config documentation

### PR DESCRIPTION
**Pull request by cehali**
# Summary
Documentation updated regarding SDK config 

Fixes: 0

## Checklist
- [x] Change is small and easy to review (please split big changes into multiple PRs)
- [x] Change is consistent with architecture
- [x] Change is consistent with test architecture
- [x] Change is consistent with naming conventions
- [x] New code is covered with tests
- [x] Tests related to old code are updated
- [x] Documentation is up to date with changes

### CODE REVIEW

File: docs/source/sdk.rst [hunk 1]
```diff
@@ -10,12 +10,23 @@ An SDK is a JS library that helps to communicate with a relayer. The SDK makes i
...
-**new UniversalLoginSDK(relayerURL, providerURL, messageOptions)**
+**new UniversalLoginSDK(relayerURL, providerURL, sdkConfig)**
 
   Parameters:
     - **relayerURL** : string - a URL address of a relayer
     - **providerURL** : string - JSON-RPC URL of an Ethereum node
-    - **messageOptions** (optional) : object - specific message options as ``gasPrice`` or ``gasLimit``
+    - **sdkConfig** (optional) : object - specific sdk options listed below:
+
+      - applicationInfo : object - detailed info about application: applicationName, type (e.g. laptop, mobile), logo (image source)
```
Comment (JustynaBroniszewska) [inline docs/source/sdk.rst]: We should add an example here: 
```
    {
        applicationName: 'Universal Login Application',
        type: 'laptop',
        logo: 'http://logo.universallogin.io'
    }
```

File: docs/source/sdk.rst [hunk 2]
```diff
@@ -10,12 +10,23 @@ An SDK is a JS library that helps to communicate with a relayer. The SDK makes i
...
+      - paymentOptions : object - specific transaction options: gasLimit, gasPrice, gasToken
```
Comment (JustynaBroniszewska) [inline docs/source/sdk.rst]: We will also need an example here

File: docs/source/sdk.rst [hunk 3]
```diff
@@ -10,12 +10,23 @@ An SDK is a JS library that helps to communicate with a relayer. The SDK makes i
...
+      - observedTokensAddresses : array of strings - addresses of tokens used
+      - observedCurrencies: array with combination of USD, ETH and DAI - shortcuts of currencies used
+      - notice : string - additional info shown on the top of React components e.g. 'Beta version'
+      - executionFactoryTick : number - perdiod of time in miliseconds which factory wait with retry if transaction failed
+      - authorizationsObserverTick : number - perdiod of time in miliseconds which last between authorization check
+      - balanceObserverTick : number - perdiod of time in miliseconds which last between balance update
+      - priceObserverTick : number - perdiod of time in miliseconds which last between price update
+
```
Comment (JustynaBroniszewska) [inline docs/source/sdk.rst]: We should provide an example of sdk config. We should also add a note that all of them are optional.

Comment (JustynaBroniszewska) [reply docs/source/sdk.rst]: We can also add our default config here.

File: docs/source/sdk.rst (hunk 1 above)
Comment (rzadp) [reply docs/source/sdk.rst]: 👍

File: docs/source/sdk.rst (hunk 3 above)
Comment (cehali) [reply docs/source/sdk.rst]: It is written that sdkConfig is optional and all parameters are in sdkConfig

File: docs/source/sdk.rst (hunk 2 above)
Comment (cehali) [reply docs/source/sdk.rst]: I will add example for the whole sdkConfig

**Pull request merged**